### PR TITLE
Synchronize tempest.conf with latest version [part 2]

### PR DESCRIPTION
**Pull request by vuntz**
The first two commits have absolutely no functional change, but are isolate to make it easier to look at the interesting bits (which are the later commits).

The last commit contains bits I'm not sure about and that we should discuss.

**untested right now**

### CODE REVIEW

File: chef/cookbooks/tempest/templates/default/tempest.conf.erb [hunk 1]
```diff
@@ -111,6 +111,33 @@ lock_path = /var/run/tempest
...
+# From tempest.config
+#
+
+# Allows test cases to create/destroy tenants and users. This option
+# requires that OpenStack Identity API admin credentials are known. If
+# false, isolated test cases and parallel execution, can still be
+# achieved configuring a list of test accounts (boolean value)
+# Deprecated group/name - [compute]/allow_tenant_isolation
+# Deprecated group/name - [orchestration]/allow_tenant_isolation
+allow_tenant_isolation = true
```
Comment (dirkmueller) [inline chef/cookbooks/tempest/templates/default/tempest.conf.erb]: it seems this causes some unstable test results.. currently reverting this part in the package to double check

Comment (dirkmueller) [inline chef/cookbooks/tempest/templates/default/tempest.conf.erb]: Yep, it fixes it. can you please revert to the default = false to make this PR mergable?

Comment (vuntz) [inline chef/cookbooks/tempest/templates/default/tempest.conf.erb]: @dirkmueller done

**Comment by dirkmueller** [comment]
+1

**Pull request merged**